### PR TITLE
[NUI] Add background span

### PR DESCRIPTION
**Pull request by shrouqsabah**
### Description of Change ###
BackgroundSpan: Span to change background color of characters

### API Changes ###
Added:
- Tizen.NUI.Text.Spans.BackgroundSpan // Class
 - Tizen.NUI.Text.Spans.BackgroundSpanBuilder // Class

### Testcases ###
Added the below to (Tizen.NUI.Devel.Tests.Ubuntu) and (Tizen.NUI.Tests):
- Tizen.NUI.Devel.Tests.PublicBackgroundSpanTest
- Tizen.NUI.Devel.Tests.PublicBackgroundSpanBuilderTest

**This PR depends on the below patches:**
- https://review.tizen.org/gerrit/c/platform/core/uifw/dali-toolkit/+/279345
- https://review.tizen.org/gerrit/c/platform/core/uifw/dali-csharp-binder/+/279346

**This PR should be preceded by the PR below:**
- https://github.com/Samsung/TizenFX/pull/4471

**Here's an example how to use it**
- https://github.com/shrouqsabah/NUI-Test/tree/spannable_test/text-spannable-backgroundspan
- https://github.com/shrouqsabah/NUI-Test

### CODE REVIEW

File: src/Tizen.NUI/src/public/Text/Spans/BackgroundSpan.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        internal BackgroundSpan(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// The fore-color in color-span.
+        /// </summary>
+        /// This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector4 BackgroundColor
```
Comment (myroot) [inline src/Tizen.NUI/src/public/Text/Spans/BackgroundSpan.cs:38]:
```suggestion
        public Color BackgroundColor
```

**Comment by myroot** [comment]
Please use Color type to express color

File: src/Tizen.NUI/src/public/Text/Spans/BackgroundSpan.cs (hunk 1 above)
Comment (shrouqsabah) [reply src/Tizen.NUI/src/public/Text/Spans/BackgroundSpan.cs]: Done

**Comment by dongsug-song** [comment]
@shrouqsabah 
Hello, 
I'm sorry for the inconvenience.
Some patches has been merged only into the master branch, so the master branch and the DevelNUI branch were conflicted.
So I did hard reset the DevelNUI branch.
Could you rebase the patch and resolve the conflict?
thank you.

the following tip could be helpful :
(execute from the work branch)
1) git fetch upstream
2) git rebase --onto=upstream/DevelNUI HEAD~1
3) git push -f
(number of commits on the work branch has to be given after '~', for this case it is 1)

**Pull request closed without merging**